js/pruebasTarea.js

```diff
@@ -0,0 +1,49 @@
+function probarValidarNombre() {
+	console.assert(
+		validarNombre(
+			"masde50caracteresmasde50caracteresmasde50caracteresmasde50caracteresmasde50caracteresmasde50caracteres"
+		) === "Mas de 50 caracteres",
+		"validarNombre deberia tirar error si hay mas de 50 caracteres, pero no lo hace"
+	);
+	console.assert(
+		validarNombre("") === "No escribiste nada",
+		"validarNombre deberia dar error si el input esta vacio."
+	);
+	console.assert(validarNombre("asd") === "", "Algo se rompio.");
+}
+
+function probarValidarCiudad() {
+	console.assert(
+		validarCiudad("") === "No se eligio ninguna ciudad",
+		"Algo se rompio"
```

Comment:
Se más específico.

Reply:
Got it! Se me siguen haciendo un poquito confusos los assert, por eso a veces no se bien que poner.